rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1626,6 +1627,41 @@ def test_remove_blockchain_account_with_tags_removes_mapping(rotkehlchen_api_ser
     assert query[0][1] == 'desktop'
 
 
+def test_add_btc_blockchain_account_ens_domain(rotkehlchen_api_server):
```

Comment:
can you also please parametrize this test to have 0 ethereum accounts so that nothing is queried for ethereum? 
You do that by:

`@pytest.mark.parametrize('number_of_eth_accounts', [0])`

Also you can also mock all BTC queries just like we do in other tests in this file. We don't want ANY queries to actually run. They are wasting our testing time. All you want to test is that the final btc address makes it in the DB.